tokens/staking/stake.md

```diff
@@ -119,102 +99,161 @@ First, you need to get the `candidateInfo`, which will contain the delegator cou
- 4. (Optional) Click the save icon and set a name for the code snippet, for example, **Get existing delegations**. This will save the code snippet locally
- 5. To execute the code, click on the run button
- 6. Copy the result as you'll need it when initiating a delegation
+Head to the **Developer** tab and click on **JavaScript**. Then take the following steps:
+
+ 1. Copy the code from the previous snippet and paste it inside the code editor box 
+ 2. (Optional) Click the save icon and set a name for the code snippet, for example, **Get delegation count**. This will save the code snippet locally
+ 3. To execute the code, click on the run button
+ 4. Copy the result as you'll need it when initiating a delegation
 
-![Get existing delegation count](/images/tokens/staking/stake/stake-13.png)
+![Get existing delegation count](/images/tokens/staking/stake/new/stake-5.png)
 
-### Staking your Tokens {: #staking-your-tokens }
+### Stake your Tokens {: #staking-your-tokens }
 
 To access staking features, you need to use the Polkadot.js Apps interface. To do so, you need to import/create an Ethereum-style account first (H160 address), which you can do by following the [Creating or Importing an H160 Account](/tokens/connect/polkadotjs/#creating-or-importing-an-h160-account){target=_blank} section of the Polkadot.js guide.
 
 For this example, an account was imported and named with a super original name: Alice. Alice's address is `0xf24FF3a9CF04c71Dbc94D0b566f7A27B94566cac`.
 
-Currently, everything related to staking needs to be accessed via the **Extrinsics** menu, under the **Developer** tab:
-
-![Staking Account](/images/tokens/staking/stake/stake-5.png)
-
-To delegate a candidate, provide the following information:
+To delegate a candidate and set up auto-compounding for your staking rewards, take the following steps:
 
  1. Select the account from which you want to stake your tokens
  2. Choose the **parachainStaking** pallet
- 3. Choose the **delegate** extrinsic
+ 3. Choose the **delegateWithAutoCompound** extrinsic
  4. Set the candidate's address to delegate. In this case, it is set to `{{ networks.moonbase.staking.candidates.address1 }}`
  5. Set the number of tokens you want to stake
- 6. Input the `candidateDelegationCount` you [retrieved previously from querying `candidateInfo`](#get-the-candidate-delegation-count)
- 7. Input the `delegationCount` [you retrieved from the JavaScript console](#get-your-number-of-existing-delegations). This is `0` if you haven't yet delegated a candidate
- 8. Click the **Submit Transaction** button and sign the transaction
+ 6. Set the percentage of rewards to auto-compound by entering a number 0-100
+ 7. Input the `candidateDelegationCount` you [retrieved previously from querying `candidateInfo`](#get-the-candidate-delegation-count)
+ 8. Input the `candidateAutoCompoundingDelegationCount` you [retrieved previously from querying `autoCompoundingDelegations`](#gget-candidate-auto-compounding-count)
```

Comment:
(#gget-candidate-auto-compounding-count) should be (#get-candidate-auto-compounding-count). There's a double "g" in the beginning.